### ISSUE TITLE
Invalidate trace cache @ step 55651: expected module 319, but got module 636

### ISSUE THREAD

**Issue by SharonJin422** [huggingface/open-r1#387]
zero3
`compute_environment: LOCAL_MACHINE

debug: false
deepspeed_config:
  deepspeed_multinode_launcher: standard
  offload_optimizer_device: none
  offload_param_device: none
  zero3_init_flag: true
  zero3_save_16bit_model: true
  zero_stage: 3
distributed_type: DEEPSPEED
downcast_bf16: 'no'
machine_rank: 0
main_training_function: main
mixed_precision: bf16
num_machines: 1
num_processes: 2
rdzv_backend: static
same_network: true
tpu_env: []
tpu_use_cluster: false
tpu_use_sudo: false
use_cpu: false`

config

`model_name_or_path: Qwen2.5-0.5B-Instruct
model_revision: main
torch_dtype: bfloat16
attn_implementation: flash_attention_2

# Data training arguments
dataset_name: data/AI-MO/NuminaMath-TIR
dataset_configs:
- all
# Num processes is less by 1 as vLLM is using 1 GPU
num_processes: 2

# GRPO trainer config
bf16: true
use_vllm: false
vllm_device: auto
vllm_gpu_memory_utilization: 0.7
do_eval: false
eval_strategy: "no"
eval_steps: 100
gradient_accumulation_steps: 16
gradient_checkpointing: true
gradient_checkpointing_kwargs:
  use_reentrant: false
learning_rate: 2.0e-05
log_level: info
logging_steps: 50
logging_strategy: steps
lr_scheduler_type: cosine
max_prompt_length: 512
max_completion_length: 1024
max_steps: -1
num_generations: 2
num_train_epochs: 1
output_dir: Qwen2.5-0.5B-Open-R1-Distill
overwrite_output_dir: true
per_device_eval_batch_size: 32
per_device_train_batch_size: 2
push_to_hub: false
report_to:
- none
save_strategy: "no"
seed: 42
warmup_ratio: 0.1`

grpo training is stuck and there are repeated information as follows, and no loss logs updated.
`Invalidate trace cache @ step 55651: expected module 319, but got module 636`

**Comment by youyc22** [comment]
Have you solved your problem? I met the same problem

**Comment by SharonJin422** [comment]
I set use_vllm as True and solved, but got another error…

| |
金小蓉
|
|
***@***.***
|
---- Replied Message ----
| From | Yichen ***@***.***> |
| Date | 02/22/2025 00:03 |
| To | huggingface/open-r1 ***@***.***> |
| Cc | SharonJin422 ***@***.***>,
State ***@***.***> |
| Subject | Re: [huggingface/open-r1] Invalidate trace cache @ step 55651: expected module 319, but got module 636 (Issue #387) |

Have you solved your problem? I met the same problem

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you modified the open/close state.Message ID: ***@***.***>

youyc22 left a comment (huggingface/open-r1#387)

Have you solved your problem? I met the same problem
